app/src/main/res/layout/home_fragmment.xml

```diff
@@ -25,7 +25,7 @@
     <ListView
         android:id="@+id/homeList"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="fill_parent"
```

Comment:
It should most definitely be fill_parent. Wrap_content gives no information on how big the ListView needs to be, meaning it has to query from ES, pull items, count items, size accordingly with the screen, query again once it finishes sizing, pull, and then display. With fill_parent, our listview knows exactly how big it has to be from MainScreen.java, and it simply has to query ES, pull, count, and display. If you check the runtime of our program, this simple change more than halves the loading delay.